libs/net/dal/Migrations/20230707155830_1.0.42.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using TNO.DAL;
+
+#nullable disable
+
+namespace TNO.DAL.Migrations
+{
+    /// <inheritdoc />
+    public partial class _1042 : SeedMigration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
```

Comment:
I do not think you generated this as it's missing the creation of the `system_message` table.  You can use the PreUp scripts to copy data to a temporary table.  Then use the PostUp scripts to update the new table.

Reply:
Updated, and we might not need a temporary table in this case, thanks